ani-cli

```diff
@@ -193,15 +193,19 @@ get_episode_url() {
     links=$(cat "$cache_dir"/* | sed 's|^Mp4-||g;/http/!d;/Alt/d' | sort -g -r -s)
     rm -r "$cache_dir"
     episode=$(select_quality "$quality")
-    [ -z "$episode" ] && die "Episode not released!"
+    if echo "$ep_list" | grep -q "^$ep_no$"; then
```

Comment:
Please use printf, it's the standard for the script. echo's behavior is implementations-specific and not standardized in POSIX

Reply:
Thanks for the feedback! I have changed to using printf. I am not very familiar with what all is POSIX compliant and just assumed something so standard would be. This is tested working in my same test cases, so I think we're good on that front.

Note: in the iina function there is an echo as well. It's mac-specific, so it may not matter, and I don't have one to test with. I just caught it when searching for echo in neovim.